contrib/cirrus/lib.sh

```diff
@@ -290,3 +290,42 @@ remove_packaged_podman_files() {
     # Be super extra sure and careful vs performant and completely safe
     sync && echo 3 > /proc/sys/vm/drop_caches || true
 }
+
+# Execute make localbenchmarks in $CIRRUS_WORKING_DIR/data
+# for preserving as a task artifact.
+localbenchmarks() {
+    local datadir
+    req_env_vars DISTRO_NV PODBIN_NAME PRIV_NAME TEST_ENVIRON TEST_FLAVOR
+    req_env_vars VM_IMAGE_NAME EC2_INST_TYPE
+
+    datadir=$CIRRUS_WORKING_DIR/data
+    mkdir -p $datadir
+
+    # Most of these are defined by cirrus and it's YAML.
+    # shellcheck disable=SC2154,SC2153
+    cat <<EOF>$datadir/benchmarks.env
+# Metadata reflecting benchmark basis.
+# Intended for machine consumption.
+CIRRUS_BASE_BRANCH=$CIRRUS_BASE_BRANCH
+CIRRUS_BRANCH=$CIRRUS_BRANCH
+CIRRUS_BUILD_ID=$CIRRUS_BUILD_ID
+CIRRUS_CHANGE_IN_REPO=$CIRRUS_CHANGE_IN_REPO
+CIRRUS_CRON=$CIRRUS_CRON
+CIRRUS_PR=$CIRRUS_PR
+CIRRUS_TASK_ID=$CIRRUS_TASK_ID
+CPUTOTAL=$(grep -ce '^processor')
```

Comment:
Missing `/proc/cpuinfo`. And, I think you missed my comment about duplication.

Reply:
oh oops.  Yeah, my carlessness-o-meter is maxed out today, sorry.  Only cure is more sleep :disappointed: